### PR TITLE
Use `Package` instead of `PackageSystem`

### PR DESCRIPTION
**Pull request by imobachgs**
Adapt to `Package` and `PackageSystem` changes. See https://github.com/yast/yast-yast2/pull/1219 for further information.

### CODE REVIEW

**Review by dgdavid — APPROVED** [review]

File: library/network/src/lib/network/susefirewall.rb [hunk 1]
```diff
@@ -554,10 +554,10 @@ def SuSEFirewallIsInstalled
       return true if @needed_packages_installed
 
       if Mode.normal
-        @needed_packages_installed = PackageSystem.CheckAndInstallPackages([@FIREWALL_PACKAGE])
+        @needed_packages_installed = Package.CheckAndInstallPackages([@FIREWALL_PACKAGE])
         log.info "CheckAndInstallPackages -> #{@needed_packages_installed}"
       else
-        @needed_packages_installed = PackageSystem.Installed(@FIREWALL_PACKAGE)
+        @needed_packages_installed = Package.Installed(@FIREWALL_PACKAGE)
```
Comment (jreidinger) [inline library/network/src/lib/network/susefirewall.rb:560]: good example that we should finally clean susefirewall, as it is dead technology and we use only firewalld.

File: library/network/src/modules/DnsServerAPI.pm [hunk 2]
```diff
@@ -657,15 +657,15 @@ sub UnquoteString {
 sub Init {
     if (Mode->test ()) {
       $package_installed = 1;
-      y2milestone("TestMode -> PackageSystem->Installed: ", $package_installed);
+      y2milestone("TestMode -> Package->Installed: ", $package_installed);
```
Comment (jreidinger) [inline library/network/src/modules/DnsServerAPI.pm:660]: dead code. old testsuite is dead and so mode->test

File: library/packages/src/include/packages/common.rb [hunk 3]
```diff
@@ -150,6 +150,7 @@ def RemoveAllMsg(packages, message)
     end
 
     def Install(package)
+      log.info "DEBUG: call Install(#{package.inspect})"
```
Comment (jreidinger) [inline library/packages/src/include/packages/common.rb:153]: this looks like leftover. Either use really log.debug or remove that DEBUG prefix.

File: library/network/src/lib/network/susefirewall.rb (hunk 1 above)
Comment (imobachgs) [reply library/network/src/lib/network/susefirewall.rb:560]: You are right, but I do not plan to do it in this PR.

**Review by jreidinger — APPROVED** [review]

**Pull request merged**